utils/riff/colors.js

```diff
@@ -85,24 +85,24 @@ const PeerColors = [
  */
 const networkGraphNodeColors = {
     you: {
-        backgroundColor: '#8e7cc3', // purple
+        backgroundColor: Colors.brightPlum,
         fontColor: Colors.white,
     },
     course: {
-        backgroundColor: '#6aa84f', // green
+        backgroundColor: Colors.sage,
         fontColor: Colors.black,
     },
     learningGroupColors: [
         {
-            backgroundColor: '#2b78e4', // blue
+            backgroundColor: Colors.turkoise,
```

Comment:
i know you didn't introduce this and fixing it may be beyond the scope of this task, but i can't *not* mention it... it's spelled `turquoise`

Reply:
LOL, yeah, that is my mea culpa, and it still doesn't "look" wrong to me.

Reply:
Fixed.

Reply:
amazing